example.py

```diff
@@ -0,0 +1,51 @@
+import yaml
+import parastell
+import numpy as np
+
+# Load configuration from YAML file
+config_file = 'config.yaml'
+with open(config_file, 'r') as file:
+    config = yaml.safe_load(file)
+
+# Access radial_build configurations
+radial_build = config['radial_build']
+phi_list = radial_build['phi_list']
+theta_list = radial_build['theta_list']
+repeat = radial_build['repeated'] 
+wall_s = radial_build['wall_s'] 
+
+# Correctly access stellarator_parameters
+stellarator_parameters = config['stellarator_parameters']
+num_phi = stellarator_parameters['num_phi']
+num_theta = stellarator_parameters['num_theta']
+
+# Prepare the build dictionary according to your requirements
+build = {
+    'phi_list': phi_list,
+    'theta_list': theta_list,
+    'wall_s': wall_s,
+    'radial_build': radial_build['components'],
+    'components': {}
+}
```

Comment:
Can't you just define this as `config['radial_build']` and then process the details into an NumPy array?

Reply:
It seemed like it made more sense to have similar parameters grouped together in the yaml file. Even though they are apart of different inputs to parastell. I wasn't entirely sure if we'd be able to group the inputs for parastell together. And I didn't want to assign values to build that wouldn't be called elsewhere. But I can separate out variables from the radial_build in the yaml file like wall_s and repeated.

Reply:
Then I can just set build as config['radial_build'] without the issues.

Reply:
I think this may be worth a conversation as a group to balance the user experience vs the code benefits